<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckpointWidget</class>
 <widget class="QWidget" name="CheckpointWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Model Checkpoints</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Checkpoint Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="checkpointNameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="saveCheckpointButton">
          <property name="text">
           <string>Save Checkpoint</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QListView" name="checkpointsListView"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="loadCheckpointButton">
        <property name="text">
         <string>Load Checkpoint</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
